### PR TITLE
update invite info

### PR DESCRIPTION
**Pull request by nataliejeremy**
removed text that stated the user has to already have a Pantheon account, and made minor edits @bmackinney please review and +1

### CODE REVIEW

File: source/docs/articles/organizations/managing-sites-and-teams-with-the-organization-dashboard.md [hunk 1]
```diff
@@ -9,37 +9,34 @@ The Organization Dashboard is where you manage all your sites in a single locati
...
-<strong>Note</strong>: Only existing Pantheon users may be added to your organization. If you want all new users whose email addresses are on your company's domain to automatically be added to your org upon registration, please contact your account executive.
-</div>
 1. Click on the **People tab**.
 2. Click **Add user**.
 3. Enter the user's email address.
 4. Choose the user's role.
 5. Click **Add user**.
 
-If the user does not exist on Pantheon, please direct them to  [https://dashboard.pantheon.io/register](https://dashboard.pantheon.io/register) and add them when their account is active. If your organization is set up to automatically add users with email addresses on your domain, simply ask them to create accounts on Pantheon with the correct email address.
+If the person does not yet have a Pantheon account, they will be sent an email with an invite to become a team member on the site. Once they have successfully signed up for an account, they will be automatically added to the team.
```
Comment (bmackinney) [inline source/docs/articles/organizations/managing-sites-and-teams-with-the-organization-dashboard.md]: site -> organization. Also, do we want to replace "signed up for" with "created"?

Comment (nataliejeremy) [inline source/docs/articles/organizations/managing-sites-and-teams-with-the-organization-dashboard.md]: I took this from Team Management, so I can update it on that page too.

**Comment by nataliejeremy** [comment]
@bmackinney I updated that. Can you +1?

**Comment by rachelwhitton** [comment]
+1

**Pull request merged**